chpl/chpl-service/src/main/java/gov/healthit/chpl/manager/impl/DeveloperManagerImpl.java

```diff
@@ -427,22 +414,6 @@ public DeveloperDTO merge(final List<Long> developerIdsToMerge, final DeveloperD
             throw new ValidationException(getDuplicateChplProductNumberErrorMessages(duplicateChplProdNumbers), null);
         }
 
-        // check for any non-active developers and throw an error if any are found
-        for (DeveloperDTO beforeDeveloper : beforeDevelopers) {
```

Comment:
Is there an UpdateActionPermissions for Developers already in the code base that has this check? Or was part of the task to remove this security?

Reply:
Moved this check to the MergeActionPermissions.